std-bits/table/src/main/java/org/enso/table/util/ColumnMapper.java

```diff
@@ -0,0 +1,50 @@
+package org.enso.table.util;
+
+import org.enso.table.data.table.Column;
+import org.enso.table.data.table.Table;
+import org.enso.table.error.ColumnCountMismatchException;
+import org.enso.table.error.ColumnNameMismatchException;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class ColumnMapper {
+  public static Table MapColumnsByName(Table table, String[] columnNames)
```

Comment:
```suggestion
  public static Table mapColumnsByName(Table table, String[] columnNames)
```

shouldn't a function name start with lowercase?